frontend/lib/identification/activation_workflow/activation_code_scanner_page.dart

```diff
@@ -100,4 +72,51 @@ class ActivationCodeScannerPage extends StatelessWidget {
       await showError("Ein unerwarteter Fehler ist aufgetreten.");
     }
   }
+
+  Future<void> _activateCode(
+    BuildContext context,
+    DynamicActivationCode activationCode,
+  ) async {
+    final client = GraphQLProvider.of(context).value;
+    final projectId = Configuration.of(context).projectId;
+    final provider = Provider.of<UserCodeModel>(context, listen: false);
+    final activationSecretBase64 = const Base64Encoder().convert(activationCode.activationSecret);
+    final cardInfoBase64 = activationCode.info.hash(activationCode.pepper);
+
+    final activationResult = await activateCode(
+      client: client,
+      projectId: projectId,
+      activationSecretBase64: activationSecretBase64,
+      cardInfoHashBase64: cardInfoBase64,
+      overwriteExisting: false,
```

Comment:
I think until we implement the ping-pong, we want to always overwrite the existing activation?
```suggestion
      overwriteExisting: true,
```

Reply:
Sure. I'm fine with both.